assess/decorators/anomalydecorator.py

```diff
@@ -81,8 +81,8 @@ def _update(self, decorator):
     def _finished_range(self):
         result = []
         for i, ensemble_result in enumerate(self._tmp_prototype_counts):
-            result.append(zip([0 for _ in ensemble_result],
-                              [max_value * self._percentage for max_value in ensemble_result]))
+            result.append(list(zip([0 for _ in ensemble_result],
+                              [max_value * self._percentage for max_value in ensemble_result])))
```

Comment:
I think this could be written without the ``zip``:

```suggestion
                     result.append([(0, max_value * self._percentage) for max_value in ensemble_result])
```